source/whats-new.txt

```diff
@@ -37,43 +37,43 @@ Learn what's new in:
 
 .. _version-5.0:
 
-What's New in 5.0
------------------
-
-.. warning:: Breaking Changes in v5.0
-
-   This driver version introduces breaking changes. For a list of these changes, see 
-   the :ref:`Version 5.0 Breaking Changes section <java-breaking-changes-v5.0>` in the 
-   Upgrade guide.
-
-The 5.0 driver release introduces the following features:
-
-- Consistent computation of the ``ClusterConnectionMode`` type after
-  instantiating a ``ClusterSettings``. The connection mode defaults to
-  ``SINGLE`` if you specify one host and do not specify a replica set
-  name, or it defaults to ``MULTIPLE`` if you specify more than one
-  host.
-
-- ``BsonDecimal128`` values respond to method calls in the same way as Java
-  ``Decimal128`` values. In particular, the error responses for the
-  ``isNumber()`` and ``asNumber()`` methods match the Java responses for
-  equivalent ``Decimal128`` values.
-
-- The ``getElapsedTime()`` method on ``com.mongodb.event.ConnectionReadyEvent``
-  includes the time taken to deliver the ``ConnectionCreatedEvent``. That is,
-  the time returned includes the duration of the
-  ``com.mongodb.event.ConnectionPoolListener.connectionCreated()`` method.
-  
-  The ``getElapsedTime()`` methods on
-  ``com.mongodb.event.ConnectionCheckedOutFailedEvent`` and
-  ``com.mongodb.event.ConnectionCheckedOutEvent`` include the time taken to
-  deliver the ``com.mongodb.event.ConnectionCheckOutStartedEvent``. That is, the
-  time returned includes the duration of the
-  ``com.mongodb.eventConnectionPoolListener.connectionCheckOutStarted()`` method.
-
-- The ``org.mongodb.scala.Observable.completeWithUnit()`` method is deprecated.
-  This relates to a :ref:`breaking change about Observables in this version
-  <java-breaking-changes-v5.0-observables>`.
+  What's New in 5.0
+  -----------------
+
+  .. warning:: Breaking Changes in v5.0
+
+    This driver version introduces breaking changes. For a list of these changes, see 
+    the :ref:`Version 5.0 Breaking Changes section <java-breaking-changes-v5.0>` in the 
+    Upgrade guide.
+
+  The 5.0 driver release introduces the following features:
+
+  - Consistent computation of the ``ClusterConnectionMode`` type after
+    instantiating a ``ClusterSettings``. The connection mode defaults to
+    ``SINGLE`` if you specify one host and do not specify a replica set
+    name, or it defaults to ``MULTIPLE`` if you specify more than one
+    host.
+
+  - ``BsonDecimal128`` values respond to method calls in the same way as Java
+    ``Decimal128`` values. In particular, the error responses for the
+    ``isNumber()`` and ``asNumber()`` methods match the Java responses for
+    equivalent ``Decimal128`` values.
```

Comment:
I am not insisting on any particular way to describe the change, but the current description is not what we need. Granted, having the "Documentation Changes Summary" attribute filled on the relevant Java ticket could have been helpful.

> `BsonDecimal128` values respond to method calls in the same way as Java `Decimal128` values

- This part shouldn't be talking about all methods.
- _"Java `Decimal128` values"_ - I don't thing "Java" is meaningful here. Everything we are talking about here is "Java", and `Decimal128` values are no more "Java" than "BsonDecimal128" values (also, both classes come from the [`org.mongodb:bson`](https://central.sonatype.com/artifact/org.mongodb/bson) library). Yet here and in the next sentence there seem to be emphasis on `Decimal128` being "Java".

> In particular, the error responses for the ``isNumber()`` and ``asNumber()`` methods match the Java responses for equivalent ``Decimal128`` values.

It's weird to talk about error responses for the methods `BsonDecimal128.isNumber`/`asNumber` (and the similar ones from `Decimal128`), given that they don't error anymore.

**Following is a summary of what has changed according to my understanding:**

- The method `BsonDecimal128.isNumber` now correctly returns `true` (previously it was returning `false`).
- The method `BsonDecimal128.isNumber` now correctly returns `BsonNumber` equal the `BsonDecimal128` it was called on (previously it was throwing an exception).
- The methods `BsonDecimal128.intValue`/`longValue`/`doubleValue` now behave identically to the methods `org.bson.types.Decimal128.intValue`/`longValue`/`doubleValue` respectively.